### DISCUSSION TITLE
Export app list

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [FossifyOrg/General-Discussion#104] [Ideas]
Perhaps a fork could be maintained
Android 14,  can't be installed, Author hasn't used GitHub in 9 years
https://f-droid.org/en/packages/de.onyxbits.listmyapps/
https://github.com/onyxbits/listmyaps

[nettnikl/listmyaps](https://github.com/nettnikl/listmyaps) is the most promising fork. The [targetSDK](https://github.com/nettnikl/listmyaps/commit/1756dc17be82b8d409b2cf23e662b07e11f706db#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9) is 29.

**Comment by inson1** [comment]
@Pantyhose-X Fossify doesnt have enough resorces even for itself

**Comment by Pantyhose-X** [comment]
Maybe I should get f-droid to switch to fork
